src/microbe_stage/PatchMapGenerator.cs

```diff
@@ -8,63 +10,386 @@
 [SuppressMessage("ReSharper", "StringLiteralTypo", Justification = "Patch names aren't proper words")]
 public static class PatchMapGenerator
 {
+    // IDs for the patches in the predefined map (not the IDs of patches in the procedural map!)
+    private static readonly int CoastalId = 0;
+    private static readonly int EstuaryId = 1;
+    private static readonly int TidepoolId = 2;
+    private static readonly int EpipelagicId = 3;
+    private static readonly int MesopelagicId = 4;
+    private static readonly int BathypelagicId = 5;
+    private static readonly int AbyssopelagicId = 6;
+    private static readonly int SeafloorId = 7;
+    private static readonly int CaveId = 8;
+    private static readonly int IceShelfId = 9;
+    private static readonly int VentsId = 10;
+
     public static PatchMap Generate(WorldGenerationSettings settings, Species defaultSpecies, Random? random = null)
     {
         var map = new PatchMap();
-
+        var predefinedMap = new PatchMap();
         random ??= new Random(settings.Seed);
 
+        // Return the classic map if settings require it, otherwise use it to draw the procedural map
+        predefinedMap = PredefinedMap(predefinedMap, TranslationServer.Translate("PANGONIAN"));
+        if (settings.MapType == WorldGenerationSettings.PatchMapType.Classic)
+        {
+            ConfigureStartingPatch(predefinedMap, settings, defaultSpecies,
+                predefinedMap.GetPatch(VentsId), predefinedMap.GetPatch(TidepoolId), random);
+            return predefinedMap;
+        }
+
         var nameGenerator = SimulationParameters.Instance.GetPatchMapNameGenerator();
 
-        string areaName = string.Empty;
-        switch (settings.MapType)
+        // Initialize the graphs random parameters
+        var regionCoords = new List<Vector2>();
+        int[,] graph = new int[100, 100];
+        int vertexNr = random.Next(6, 10);
```

Comment:
```suggestion
        int vertexNumber = random.Next(6, 10);
```

No, non approved abbreviations please